backendImpl/src/main/java/com/virtuslab/gitmachete/backend/impl/GitMacheteRepository.java

```diff
@@ -75,44 +76,37 @@ public class GitMacheteRepository implements IGitMacheteRepository {
 
   @Override
   @UIThreadUnsafe
+  @Loggable(value = Loggable.DEBUG, prepend = true, skipArgs = true, skipResult = true)
```

Comment:
Skip args and result coz they're not printable

Reply:
`prepend`, as in also log the start of method, not just the finish

Reply:
Entry to method (if `prepend = true`):

```
2022-04-26 20:40:38,769 [  20596]   FINE - c.v.g.b.i.GitMacheteRepository - #createSnapshotForLayout(...): entered
```

Exit from method:

```
2022-04-26 20:40:39,270 [  21097]   FINE - c.v.g.b.i.GitMacheteRepository - #createSnapshotForLayout(...): ... in 504.57ms
```

I mean, as per slf4j's philosophy, the ultimate formatting comes from whatever slf4j implementation is bound in the runtime &mdash; it used to be log4j until 2021, now it's java.util.logging since IntelliJ 2022.1, curious why the change :trollface: 

What the loggers have control over is just the message (so from `#...` in this example)